code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -60,6 +61,11 @@
 		var/datum/poll_question/poll = locate(href_list["votepollref"]) in GLOB.polls
 		vote_on_poll_handler(poll, href_list)
 
+/mob/dead/new_player/get_status_tab_items()
+	. = ..()
+	if(!SSticker.HasRoundStarted()) //only show this when the round hasn't started yet
+		. += "Readiness status: [ready ? "Ready" : "Not Ready"]"
```

Comment:
```suggestion
		. += "[ready ? "" : "<b>Not</b>"] Readied Up!"
```

just mentioning this as a possibility since you asked. it does break the format of that whole window, but I think that's okay because it's pretty important to stick out.

Reply:
i think it's fine to emphasize this with bold text, but i'm not sure if having it without any prefix is ok

Reply:
probably up to you then. i also forget if bold text works in the stat panel or not, odds are that it doesn't